webpack.config.js

```diff
@@ -145,34 +156,44 @@ module.exports = ( env ) => {
       }),
```

Comment:
please add ifProdOrSite here as well but without true false hell and add no callback support to the helper